### PR TITLE
v1.7.0 with release notes

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by nayeemrmn** [comment]
1.7.0 should be removed from the list of upcoming releases in https://github.com/denoland/deno/blob/a2550ab6786bf7f86db4c037d005da9f11ab64b9/docs/contributing/release_schedule.md#release-schedule as well.

**Review by piscisaureus — APPROVED** [review]
LGTM

File: Releases.md [hunk 1]
```diff
@@ -6,6 +6,90 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at:
 https://github.com/denoland/deno_install
 
+### 1.7.0 / 2021.01.19
+
+- BREAKING(unstable): Use hosts for net allowlists (#8845)
+- BREAKING(unstable): remove CreateHttpClientOptions.caFile (#8928)
+- feat(installer): Add support for MSYS on Windows (#8932)
```
Comment (piscisaureus) [inline Releases.md:13]: Nit: "install"/"installer" not used consistently. Don't hold up the release for this.

File: Releases.md [hunk 2]
```diff
@@ -6,6 +6,90 @@ https://github.com/denoland/deno/releases
...
+  #9139)
+- feat: Add WorkerOptions interface to type declarations (#9147)
+- feat: Add configurable permissions for Workers (#8215)
+- feat: Standalone lite binaries and cross compilation (#9141)
+- feat: add --location=<href> and globalThis.location (#7369)
+- feat: add global tls session cache (#8877)
+- feat: add markdown support to deno fmt (#8887)
+- feat: add utf-16 and big5 to TextEncoder/TextDecoder (#8108)
+- feat: denort binary (#9041)
+- feat: stabilize Deno.shutdown() and Conn#closeWrite()(#9181)
```
Comment (piscisaureus) [inline Releases.md:25]: Nit: missing space. Don't hold up the release for this.

**Pull request merged**